support/github-to-jira-synchronization/liferay-frontend-projects.md

```diff
@@ -0,0 +1,164 @@
+# liferay/liferay-frontend-projects GitHub to JIRA syncrhonization
+
+This document describes the workflow between the [Clay](https://github.com/liferay/liferay-frontend-projects) project in GitHub and issues.liferay.com.
```

Comment:
it should be "...workflow between the **liferay-frontend-project** ..." instead of Clay

Reply:
> it should be "...workflow between the liferay-frontend-project ..." instead of Clay

![mimimi](https://user-images.githubusercontent.com/905006/118645291-a55aae80-b7d6-11eb-821c-8a0b02cc1cce.gif)

Reply:
(I've pushed the change as well, just hiding in the same commit so you can't easily know if I did change it or not :p)

Reply:
![magic trick](https://user-images.githubusercontent.com/74542632/118653245-9c260d80-b7e7-11eb-8af7-f89f8c5d45d8.gif)